services/app-web/src/components/Header/PageHeadingRow/PageHeadingRow.tsx

```diff
@@ -82,6 +82,6 @@ export const PageHeadingRow = ({
     } else if (loggedInUser.__typename === 'StateUser') {
         return <StateUserRow user={loggedInUser} heading={heading} />
     } else {
-        return <h1>Programming Error: Unkown User Type: {loggedInUser}</h1>
+        return <h1>{`Programming Error: Unkown User Type: ${loggedInUser}`}</h1>
```

Comment:
why do we have to interpolate this? I thought JSX would work with the old version. Is this a new React change?

Reply:
This does look like new behavior, though I haven't tracked down the root cause.  The new error is `This JSX tag's 'children' prop expects a single child of type 'ReactNode', but multiple children were provided.`  So I turned it into a single thing.